specs/protocol/holocene/predeploys.md

```diff
@@ -0,0 +1,252 @@
+# Overview
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Constants](#constants)
+- [Predeploys](#predeploys)
+  - [L1Block](#l1block)
+    - [Storage](#storage)
+    - [Interface](#interface)
+      - [`setL1BlockValuesHolocene`](#setl1blockvaluesholocene)
+      - [`eip1559Elasticity`](#eip1559elasticity)
+      - [`eip1559Denominator`](#eip1559denominator)
+      - [`setConfig`](#setconfig)
+      - [`baseFeeVaultConfig`](#basefeevaultconfig)
+      - [`sequencerFeeVaultConfig`](#sequencerfeevaultconfig)
+      - [`l1FeeVaultConfig`](#l1feevaultconfig)
+      - [`l1CrossDomainMessenger`](#l1crossdomainmessenger)
+      - [`l1StandardBridge`](#l1standardbridge)
+      - [`l1ERC721Bridge`](#l1erc721bridge)
+      - [`remoteChainId`](#remotechainid)
+  - [FeeVault](#feevault)
+    - [Interface](#interface-1)
+      - [`config`](#config)
+  - [L2CrossDomainMessenger](#l2crossdomainmessenger)
+    - [Interface](#interface-2)
+  - [L2ERC721Bridge](#l2erc721bridge)
+    - [Interface](#interface-3)
+  - [L2StandardBridge](#l2standardbridge)
```

Comment:
This is missing sections for `OptimismMintableERC721Factory` and `L2ERC721Bridge`